### ISSUE TITLE
Ternary expression options

### ISSUE THREAD

**Issue by leeallen337** [smashingboxes/eslint-config-smashing-boxes#6]
## Multiline Ternary
I'm proposing that we change the `multiline-ternary` setting found [here](https://github.com/smashingboxes/eslint-config-smashing-boxes/blob/master/defaults.js#L207) from `off` to `always-multiline`

See the documentation [here](https://eslint.org/docs/rules/multiline-ternary) and the specific setting [here](https://eslint.org/docs/rules/multiline-ternary#always-multiline)

## Nested Ternary

I'm proposing we change the `no-nested-ternary` setting found [here](https://github.com/smashingboxes/eslint-config-smashing-boxes/blob/master/defaults.js#L229) from `error` to `off`. 

See the documentation [here](https://eslint.org/docs/rules/no-nested-ternary)

## This changes would most likely necessitate a major version bump.

**Comment by OzoneTank** [comment]
I would vote we also change
https://github.com/smashingboxes/eslint-config-smashing-boxes/blob/master/defaults.js#L245
to something like
`"operator-linebreak": ["error", "after",
  { "overrides": { "?": "before", ":": "before" } }
],  `
I think

**Comment by leeallen337** [comment]
@OzoneTank Can you please make a separate issue for that? While I understand that the rule your suggesting targets ternaries specifically, the rule as a whole encompasses a lot more than ternaries so it might be better to have it's own issue 👍

**Comment by keeganbrown** [comment]
TBH, I'd question the need for this to require a major version bump. If the linter was going to start spitting out errors where it wasn't before, then definitely, but I don't think that applies here.

**Comment by leeallen337** [comment]
@keeganbrown Hm good points. I think changing `no-nested-ternaries` from `error` to `off` wouldn't through errors for anyone since no one was doing nested ternaries previously and then we would start allowing them.

However, I do wonder about the `always-multiline` rule. I think most people break their ternaries into multiple lines when it won't fit on a single line but there is the possibility of people writing something like
```javascript
expression ?
  true : false
```
which would through an error, when the below would be correct under `always-multiline`
```javascript
expression ?
  true :
  false

// OR

expression ? true : false
```
What do you think?

**Comment by OzoneTank** [comment]
I agree with @keeganbrown this feels like low harm since it will be producing less errors.

**Comment by keeganbrown** [comment]
@leeallen337 yeah, I see your point on always multiline. Ideally I'd only like to force multiline ternaries if they are nested.
